.evg.yml

```diff
@@ -531,17 +531,26 @@ functions:
         bucket: translators-connectors-releases
 
   "build jdbc driver":
-    command: shell.exec
-    type: test
-    params:
-      shell: bash
-      working_dir: mongo-jdbc-driver
-      script: |
-        ${PREPARE_SHELL}
-        if [[ "${triggered_by_git_tag}" != "" ]]; then
-          EXTRA_PROP="-PisTagTriggered"
-        fi
-        ./gradlew -Dorg.gradle.java.home=${JAVA_HOME} $EXTRA_PROP clean -x test -x integrationTest spotlessApply build shadowjar --rerun-tasks
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongo-jdbc-driver
+        script: |
+          #./gradlew -Dorg.gradle.java.home=${JAVA_HOME} generateLicenseReport
+          ./gradlew generateLicenseReport
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongo-jdbc-driver
+        script: |
+          ${PREPARE_SHELL}
+          if [[ "${triggered_by_git_tag}" != "" ]]; then
+            EXTRA_PROP="-PisTagTriggered"
+          fi
+          #./gradlew -Dorg.gradle.java.home=${JAVA_HOME} $EXTRA_PROP clean -x test -x integrationTest spotlessApply build shadowjar --rerun-tasks
+          ./gradlew $EXTRA_PROP clean -x test -x integrationTest spotlessApply build shadowjar --rerun-tasks
```

Comment:
I believe that you need to keep the JAVA HOME property to make use of the toolchain and avoid issue on hosts where JAVA_HOME is not set.